examples/GraphQL.java

```diff
@@ -0,0 +1,37 @@
+import com.indico.IndicoClient;
+import com.indico.IndicoConfig;
+import com.indico.request.GraphQLRequest;
+import org.json.JSONObject;
+import java.io.IOException;
+
+public class GraphQL {
+
+    public static void main(String args[]) throws IOException {
+        IndicoConfig config = new IndicoConfig.Builder()
+                .host("dev.indico.io")
+                .tokenPath("__TOKEN_PATH__")
+                .build();
+
+        try (IndicoClient client = new IndicoClient(config)) {
+            GraphQLRequest request = indico.graphQLRequest();
+            String query
+                    = "query GetDatasets {\n"
+                    + "    datasets {\n"
+                    + "        id\n"
+                    + "        name\n"
+                    + "        status\n"
+                    + "        rowCount\n"
+                    + "        numModelGroups\n"
+                    + "        modelGroups {\n"
+                    + "            id\n"
+                    + "        }\n"
+                    + "    }\n"
+                    + "}";
+
+            JSONObject response = request.query(query).operationName("GetDatasets").call();
```

Comment:
operationName can be removed here it's not used for anything.

Reply:
Fixed. Made Optional.